elliott/elliottlib/cli/verify_cdn_push_cli.py

```
@@ -0,0 +1,261 @@
+        LOGGER.error("Advisory %s (%s): error checking push status: %s", advisory_id, impetus, e)
+        result.error = str(e)
+
+    return result
+
+
+async def check_blocking_advisories(api: AsyncErrataAPI, advisory_id: int, do_push: bool) -> list[AdvisoryPushResult]:
+    try:
+        raw = await api.get_advisory(advisory_id)
+        erratum_data = next(iter(raw.get("errata", {}).values()), {})
+        blocking_ids = erratum_data.get("blocking_advisories", [])
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)